doc/api/process.markdown

```diff
@@ -903,6 +913,38 @@ a diff reading, useful for benchmarks and measuring intervals:
     }, 1000);
 
 
+## process.send(message[, sendHandle])
+
+* `message` {Object}
+* `sendHandle` {Handle object}
+
+When io.js is spawned with an IPC channel attached, it can send messages to its
+parent process using `process.send()`. Each will be received as a
+['message'](child_process.html#child_process_event_message)
+event on the parent's ChildProcess object.
+
+If io.js was not spawned with an IPC channel, `process.send()` will be undefined.
+
+
+## process.disconnect()
+
+Close the IPC channel to the parent process, allowing this child to exit
+gracefully once there are no other connections keeping it alive.
+
+Identical to the parent process's
+[ChildProcess.disconnect()](child_process.html#child_process_child_disconnect).
+
+If io.js was not spawned with an IPC channel, `process.disconnect()` will be
+undefined.
+
+
+### child.connected
+
+* {Boolean} Set to false after `.disconnect` is called
```

Comment:
This looks kind of weird with nothing before the dot. Might want to add `()` on the end too.